canonical-websites/jp.ubuntu.com
# Migration to flask

## Done
Migrates the website from Django to Flask.

Fixes https://github.com/ubuntudesign/base-squad/issues/354 , https://github.com/ubuntudesign/base-squad/issues/353 , https://github.com/ubuntudesign/base-squad/issues/352

**The issues are ordered into commits in this PR to make review easier!**

## QA
- `./run` and compare that the website looks identical to https://jp.ubuntu.com

In addition this small script will do requests to all available URLs on both the local and production version and run a diff on them. There should be no differences between them.
```bash
#! /bin/bash 

urls=("/" "/downloads" "/enterprise-support" "/ai-ml" "/enterprise-support/plans-and-pricing" "/openstack" "/kubernetes" "/contact-us" "/asdasdasdasd")

domains=("http://localhost:8012" "https://jp.ubuntu.com")

for url_index in ${!urls[*]} 
do
  for domain_index in ${!domains[*]}
  do
    new_url="${domains[$domain_index]}${urls[$url_index]}"
    echo $new_url
    # curl and diff to make sure that there are no differences
    curl $new_url > "$url_index$domain_index.txt"
    if [ $domain_index -eq 1 ]
    then
      diff  $url_index"0.txt" $url_index"1.txt"
    fi
  done
done

```
